pkg/platform/runtime/setup/setup.go

```diff
@@ -163,15 +162,15 @@ func NewWithModel(target Targeter, msgHandler Events, model ModelProvider, an an
 }
 
 // Update installs the runtime locally (or updates it if it's already partially installed)
-func (s *Setup) Update() error {
+func (s *Setup) Update(projPath, sockPath string) error {
```

Comment:
We record the project path on the runtime target, eg.:

https://github.com/ActiveState/cli/blob/ceeec3c0bbc10747cb307a60f6ffc6e487d2753a/pkg/platform/runtime/target/target.go#L79

Though this is raising an issue, which you would have faced as well; not all runtimes have activestate.yaml's. Eg offline installations and deployments.

I think maybe instead we need to record the relevant runtime information to a meta file in the runtime directory, eg.

https://github.com/ActiveState/cli/blob/b930714d64c0b3cd3485a2d012867fb2d94f6512/pkg/platform/runtime/store/store.go#L26

Then that would always be installed regardless of activestate.yaml existing.

Sorry about this, I should have foreseen this.